lib/transmuxer.js

```diff
@@ -187,8 +189,14 @@ TransportParseStream = function() {
       offset += ((payload[offset + 3] & 0x0F) << 8 | payload[offset + 4]) + 5;
     }
 
+    self.foundPMT = true;
```

Comment:
Is a separate variable for this necessary? Couldn't we just check `self.programMapTable !== undefined`?